django_filters/rest_framework/backends.py

```diff
@@ -90,8 +90,8 @@ def get_schema_fields(self, view):
                 required=False,
                 location='query',
                 schema=compat.coreschema.String(
-                    description=six.text_type(field.field.help_text)
+                    description=six.text_type(field.extra.get('help_text', ''))
```

Comment:
This should be fine since form fields don't provide their own `help_text`. Technically they can, but in practice they don't.